src/adminUser.service.ts

```diff
@@ -12,8 +12,14 @@ import AdminUser from './adminUser.entity'
 import { DuplicateEmailException } from './exceptions/userAdmin.exception'
 import { AdminUserValidationException } from './exceptions/adminUserValidation.exception'
 
+type AdminUserWithoutPassword = Omit<AdminUser, 'password'>
+
+export interface AdminUserServiceInterface extends EntitySubscriberInterface<AdminUser> {
```

Comment:
the `extends` should be in the class, not the interface. A class extending a class is different from an interface extending a class. In the second case, the extended class will be treated as an interface.